encrypted-media-respec.html

```diff
@@ -3825,25 +3825,25 @@
 
           <section id="per-origin-individualization">
             <h5>Per-Origin Individualization</h5>
-            <p>Per-<a def-id="origin"></a> individualization is performed between the <a def-id="cdm"></a> and an application-selected server and results in a per-origin identifier.
-              The process MUST be performed via these APIs and MUST NOT involve other types of communication.
-              Any <a def-id="distinctive-identifiers"></a> used MUST be origin-specific.
+            <p>Per-<a def-id="origin"></a> individualization is performed between the <a def-id="cdm"></a> and the application, including an application-selected server, and results in a per-origin identifier.
+              The process MUST be performed via these APIs and MUST NOT involve other methods of communication.
+              As with all other uses of the APIs, <a def-id="distinctive-identifiers"></a> MAY be used but indelible or non-origin-specific MUST NOT be exposed, even in encrypted form.
+              If one or more <a def-id="distinctive-identifiers"></a> is used, the resulting identifier is by definition also a <a def-id="distinctive-identifier"></a>.
             </p>
             <p>For such individualization, all message exchanges:</p>
             <ul>
-              <li><p>MUST be passed to or through the application via these APIs (message type <a def-id="message-type-individualization-request"></a>).</p></li>
+              <li><p>MUST be passed to or through the application via these APIs.</p></li>
+              <li><p>SHALL use the message type <a def-id="message-type-individualization-request"></a> for all related <a def-id="message"></a> events.</p></li>
               <li><p>MUST NOT be performed by the user agent.</p></li>
               <li><p>MUST NOT be performed directly by the CDM.</p></li>
-              <li><p>MUST NOT contain non-origin-specific per-client information, such as a <a def-id="distinctive-identifier"></a>.</p></li>
-              <li><p>As with all other uses of the APIs, responses passed to the CDM MUST NOT contain executable code.</p></li>
+              <li><p>MUST NOT contain non-origin-specific per-client information or any type of indelible or semi-permanent identifer.</p></li>
+              <li><p>MUST adhere to the <a href="#identifier-requirements">identifier requirements</a>, including exposing only values that are <a href="#per-origin-identifiers">unique per origin</a> and <a href="#allow-identifiers-cleared">clearable</a> and <a href="#encrypt-identifiers">encrypting</a> <a def-id="distinctive-identifiers"></a>.</p></li>
```

Comment:
Done. I commented it out within a TODO to address when the referenced section is refactored per #117.